modules/homepage/src/main/frontend/src/themePage/Navbars/AdminNavbarLinks.jsx

```diff
@@ -10,46 +10,151 @@
 * The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
 */
 import PropTypes from "prop-types";
-import React from "react";
+import React, { useEffect, useRef, useState } from "react";
 // @material-ui/core components
-import { Button, Hidden, IconButton, withStyles } from "@material-ui/core";
-// @material-ui/icons
-import ExitToApp from "@material-ui/icons/ExitToApp";
+import { Avatar, ClickAwayListener, Grow, Hidden, IconButton, Link, ListItemIcon, ListItemText, MenuList, MenuItem, Paper, Popper, Snackbar, withStyles } from "@material-ui/core";
```

Comment:
I find long `import` lists easier to read and subsequent changes easier to see if the imported entities are listed one per line: 

```suggestion
import {
  Avatar,
  ClickAwayListener,
  Grow,
  Hidden,
  IconButton,
  Link,
  ListItemIcon,
  ListItemText,
  MenuList,
  MenuItem,
  Paper,
  Popper,
  Snackbar,
  withStyles
} from "@material-ui/core";
```

Reply:
Applied in `1ea0029` :heavy_check_mark: